scripts/pushArgos.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-console */
+const util = require('util');
+const glob = require('fast-glob');
+const fse = require('fs-extra');
+const chunk = require('lodash/chunk');
+const childProcess = require('child_process');
+
+const execFileAsync = util.promisify(childProcess.execFile);
+
+async function exec(command, args) {
+  const options = {
+    cwd: process.cwd(),
+    env: process.env,
+    stdio: 'pipe',
+    encoding: 'utf-8',
+  };
+
+  const results = await execFileAsync(command, args, options);
+  return results.stdout;
+}
+
+const screenshotsBase = 'test/regressions/screenshots/chrome';
+const BATCH_COUNT = 2;
```

Comment:
Wouldn't it make more sense to declare the chunk **size** (And use a single term to not switch between batch and chunk. They seem equivalent in this context to me).

Otherwise we might run into timeouts again if the number of screenshots increases over time. By declaring the chunk size we automatically scale up (or down). For example, with 10 screenshots it might make more sense to upload them in one batch. But for 1000 screenshots  we would encounter the same problem as before.

I don't know what the exact problem is (number of uploads or size of uploads). If the size of the upload is the problem then we should probably create chunks of equal size not equal number of screenshots.

Reply:
Sounds great. My hypothesis is that the bottleneck is the database, inserting fewer records should help.